src/wkt/tests/timestamp.rs

```diff
@@ -99,3 +99,21 @@ fn convert_to_time() -> Result {
     assert_eq!(got, want);
     Ok(())
 }
+
```

Comment:
Do we prefer having these tests be in `wkt/src/timestamp.rs`?

Reply:
I think so, but maybe I am wrong.

The tests in this file can **only** use the public API for `Timestamp` and related trait implementations.  That makes them better tests of the behavior of the type, as they cannot use any private functions or the type internals.